katello.te

```diff
@@ -53,6 +54,18 @@ miscfiles_read_certs(websockify_t)
 
 ######################################
 #
+# Pulp
+#
+
+# Katello installer configures Celery to listen on port 5000
+ifdef(`distro_rhel6', `
```

Comment:
That's the "else" block. It works, try it :-)

This is the best way to do this. RHEL6 is special, everything else (RHEL7, modern Fedoras) are the "else" block.